### ISSUE TITLE
request: remove license prompt when adding a study

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#1039]
This info is a pain to obtain, confusing to users, and we don't use it (and never will).

e.g. I'm trying to upload the tree from http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3645.html
and (even though there is "rights and permissions" link on the website for the paper) I don't see anywhere where the license is specified with enough precision that it allows me to correctly fill in our form.

**Comment by kcranston** [comment]
I hear you about this being confusing for people. I would still like to label data CC0 if we can - what if we added an 'are you an author' question, and if yes, then asked for CC0? Otherwise, don't ask about licensing. We can also leave the question off for treebase, because we know the answer if the data comes from treebase.

**Comment by mtholder** [comment]
I don't really object to trying to categorize the license with the set of options that we have. I just think we need to make this aspect of it not a blocker to uploading. The "I am an author" and the license stuff should be on the metadata tab.
I agree that changing it to "I am an author" would have made me less grumpy because I can answer that easily.

**Comment by snacktavish** [comment]
+1

**Comment by jar398** [comment]
Regarding " we don't use it (and never will)." - of course we won't.
License and waiver information is always provided for the benefit of
*other* people - in this case those who download from phylesystem or other
parts of open tree.

Just because the license is not known in some cases, doesn't mean it's not
known in all cases.  E.g. supplementary material from BM, Hindawi, and PLoS
articles is licensed with one of the CC-BY licenses (which one depends on
the publication date) and it is very easy to determine that it is.

Not saying the UI is great, just addressing that particular bit of
reasoning.

It may be reasonable to say that if someone cares about the license/waiver
they should consult the original article, except that, as we've seen, not
all data is part of the original article. It would also be reasonable to
say that data can't be licensed, although the certainty of CC0 is very
comforting especially to institutional and corporate users.

I agree that license information could be handled on the metadata page. But
if Open Tree can play a role in harvesting a CC0 waiver, that is a help to
other people. I'm not sure that putting the CC0 business on the metadata
page is in-your-face enough.

**Comment by mtholder** [comment]
Setting the goal of being in the user's face about information that is not crucial or used by the project is a recipe for annoying users.
The vast majority of scientist want to know (1) how to get the information, and (2) whom to cite. I've never met one who cared about the license for the data, so I think we're kidding ourselves if we think this feature is helping an appreciable number of users.
Once again, I don't object to storing it and having a spot in the UI for letting users enter it, but it clearly puts a barrier at the entry point to our curation system. There is not a choice for "I don't know/care" so it is not hard for a user to feel like they can't even get the ball rolling on curating a study.

**Comment by snacktavish** [comment]
I agree, I have heard from several people that they got stuck at the point of choosing a license. We want their data, and that required prompt creates a barrier to us getting it. I think on the metadata page is fine.
